plugins/brightness/PageComponent.qml

```diff
@@ -79,12 +79,12 @@ ItemPage {
         Column {
             anchors.left: parent.left
             anchors.right: parent.right
-
-            ListItem.Standard {
+            
+            //SettingsItemTitle to be consistent with the rest of settings
+            SettingsItemTitle {
                 text: i18n.tr("Display brightness")
-                showDivider: false
             }
-
+            
```

Comment:
The only thing changed on this line is the addition of space characters. Please remove them.

The same thing occurs on line 113.

Reply:
Cool, I think I got it :)